engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/preparation/JobPreparationActorSpec.scala

```diff
@@ -1,31 +1,39 @@
 package cromwell.engine.workflow.lifecycle.execution.preparation
 
-import akka.testkit.{ImplicitSender, TestActorRef, TestProbe}
+import akka.actor.ActorSystem
+import akka.testkit.{ImplicitSender, TestActorRef, TestKit, TestProbe}
 import cromwell.core.callcaching.{CallCachingEligible, CallCachingIneligible}
 import cromwell.core.callcaching.docker.DockerHashActor.{DockerHashBackPressure, DockerHashFailedResponse, DockerHashResponseSuccess}
 import cromwell.core.callcaching.docker.{DockerHashRequest, DockerHashResult, DockerImageIdentifier, DockerImageIdentifierWithoutHash}
-import cromwell.core.{LocallyQualifiedName, TestKitSuite}
+import cromwell.core.LocallyQualifiedName
 import cromwell.engine.workflow.lifecycle.execution.preparation.CallPreparation.{BackendJobPreparationSucceeded, CallPreparationFailed, Start}
-import org.scalatest.{FlatSpecLike, Matchers}
+import cromwell.services.keyvalue.KeyValueServiceActor.{KvGet, KvKeyLookupFailed, KvPair}
+import org.scalatest.{BeforeAndAfter, FlatSpecLike, Matchers}
 import wdl4s.Declaration
 import wdl4s.values.{WdlString, WdlValue}
 
 import scala.util.{Failure, Success}
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-class JobPreparationActorSpec extends TestKitSuite with FlatSpecLike with Matchers with ImplicitSender {
+class JobPreparationActorSpec extends TestKit(ActorSystem("JobPrepActorSpecSystem")) with FlatSpecLike with Matchers with ImplicitSender with BeforeAndAfter {
```

Comment:
From the scaladoc of [`TestKit`](https://github.com/akka/akka/blob/v2.4.11/akka-testkit/src/main/scala/akka/testkit/TestKit.scala#L801-L802):

> - the ActorSystem passed into the constructor needs to be shutdown,
>    otherwise thread pools and memory will be leaked

Either use `TestKitSuite("JobPrepActorSpecSystem")`, or cleanup the thread pool after this suite finishes.